### ISSUE TITLE
New install

### ISSUE THREAD

**Issue by desmodromisk** [Ylianst/MeshCentral#4967]
Hi
I can´t access a new MeshAgent server other than by localhost ?
Firewall rules set and also port forwarding.

Best Regards
Tomas

**Comment by OutbackMatt** [comment]
can you post the contents of your config.json (without passwords)

**Comment by desmodromisk** [comment]
{

  "settings": {

    "cert": "desmo.asuscomm.com",

    "_minify": true

  }

}

 

 

Med V䮬ig H䬳ning

Tomas Berglund

0730239198

 

Från: Matt Gilchrist ***@***.***> 
Skickat: den 7 februari 2023 13:55
Till: Ylianst/MeshCentral ***@***.***>
Kopia: desmodromisk ***@***.***>; Author ***@***.***>
Ämne: Re: [Ylianst/MeshCentral] New install (Issue #4967)

 

can you post the contents of your config.json (without passwords)

**Comment by dinger1986** [comment]
as that your whole config?

**Comment by desmodromisk** [comment]
Yes as i can find in: C:\Program Files\Open Source\MeshCentral\meshcentral-data

 

 

Med V䮬ig H䬳ning

Tomas Berglund

0730239198

 

Från: dinger1986 ***@***.***> 
Skickat: den 7 februari 2023 15:51
Till: Ylianst/MeshCentral ***@***.***>
Kopia: desmodromisk ***@***.***>; Author ***@***.***>
Ämne: Re: [Ylianst/MeshCentral] New install (Issue #4967)

 

as that your whole config?

**Comment by dinger1986** [comment]
I would have expected to see something like this:

https://github.com/Ylianst/MeshCentral/blob/master/sample-config.json

**Comment by desmodromisk** [comment]
Hi

 

Yes i dont understand that too.

I had MeshCentral installed at an other pc earlier and that ran fine.

But now i only can run this installation in local host mode ?

 

 

Best Regards

Tomas Berglund

0730239198

 

Från: dinger1986 ***@***.***> 
Skickat: den 7 februari 2023 18:15
Till: Ylianst/MeshCentral ***@***.***>
Kopia: desmodromisk ***@***.***>; Author ***@***.***>
Ämne: Re: [Ylianst/MeshCentral] New install (Issue #4967)

 

I would have expected to see something like this:

https://github.com/Ylianst/MeshCentral/blob/master/sample-config.json

**Comment by OutbackMatt** [comment]
How did you install this time?

**Comment by desmodromisk** [comment]
Hello

 

Got it working after new install and removing portforvarding

For older installation. But config.json still looks odd

 

C:\Program Files\Open Source\MeshCentral\meshcentral-data

 

{

   "settings": {

      "cert": "desmo.asuscomm.com", 

      "_minify": true

   }, 

   "domains": {

      

   }

}

 

 

Med V䮬ig H䬳ning

Tomas Berglund

0730239198

 

Från: Matt Gilchrist ***@***.***> 
Skickat: den 7 februari 2023 22:35
Till: Ylianst/MeshCentral ***@***.***>
Kopia: desmodromisk ***@***.***>; Author ***@***.***>
Ämne: Re: [Ylianst/MeshCentral] New install (Issue #4967)

 

How did you install this time?
